<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptPlugins</class>
 <widget class="QWidget" name="OptPlugins" >
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OptPluginsUI</string>
  </property>
  <layout class="QVBoxLayout" stretch="0,0,1,0">
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="TextLabel2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Plugin Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_plugins">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="isOption" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_info">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>27</horstretch>
         <verstretch>27</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string></string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="version">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Version: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_loadPlugin">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Load this plugin</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Plugin Location:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_location">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>No plugin selected</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="isOption" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vboxLayout1">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
